src/app/effects/user.effects.ts

```diff
@@ -0,0 +1,48 @@
+import { Actions, Effect } from '@ngrx/effects';
+import { Injectable } from '@angular/core';
+import {
+  Notification,
+  Notifications,
+  NotificationType
+} from 'ngx-base';
```

Comment:
You are not using notification anywhere in this file. So, we don't need this import.

Reply:
Now you are using it. So this comment not relevant anymore.